vip-cache-manager/vip-cache-manager.php

```diff
@@ -215,6 +215,11 @@ public function execute_purges() {
 			return;
 		}
 
+		// Cache purging disabled, bail
+		if ( defined( 'VIP_GO_DISABLE_CACHE_PURGING' ) && true === VIP_GO_DISABLE_CACHE_PURGING ) {
```

Comment:
We should also check the core `WP_IMPORTING` constant.

Reply:
I see we check for `WP_IMPORTING` when URLs are queued. I wonder if we should also check for `VIP_GO_DISABLE_CACHE_PURGING` there as well? The benefit would be reduced memory usage since we wouldn't even put these URLs into the purge array.

https://github.com/Automattic/vip-go-mu-plugins/blob/76a6152d68542c54992ed2a462837d86a11bf060/vip-cache-manager/vip-cache-manager.php#L288

Reply:
Makes totally sense, early check added with 532c0c72